sklearn/cluster/tests/test_bicluster.py

```diff
@@ -204,7 +204,7 @@ def test_project_and_cluster():
     for mat in (data, csr_matrix(data)):
         labels = model._project_and_cluster(data, vectors,
                                             n_clusters=2)
-        assert_array_equal(labels, [0, 0, 1, 1])
+        assert_equal(v_measure_score(labels, [0, 0, 1, 1]), 1.0)
```

Comment:
Will it be better to use ``assert_almost_equal`` for floats?

Reply:
In this case sample size of a power of two, and sizes associated with each cluster are themselves powers of two, so the rational arithmetic done with floats should be exact.

I do not mind the change though. Let me know if you think it should be done.

Reply:
@oleksandr-pavlyk Unfortunately we're currently using e as log base, not 2 (likely to be fixed in the next release). See https://github.com/scikit-learn/scikit-learn/blob/5fd9e03/sklearn/metrics/cluster/supervised.py#L945. So I guess maybe it's better to use things like ``assert_almost_equal``? (We only test some environments here.)

Reply:
Thank you @qinhanmin2014, I have made the change to use `assert_almost_equal`.